docs/design/consensus.md

```diff
@@ -1,24 +1,30 @@
 ## Consensus
 
-The consensus protocol in FTLib acts as a shadow precondition for any collective communication operations. Any changes from the consensus protocol will reset the initialization flag of FTLib to `False`, deterring any communication operations after the rebuild procedure returns success.
+The consensus protocol in FTLib acts as a shadow precondition for any collective communication operations. Any changes from the consensus protocol will reset the `_initialized` flag of FTLib to `False`, deterring any communication operations after the rebuild procedure returns success. Under passive or active mode, such change takes place immediately or eventually (when FTLib check the consensus next time).
 
-A member list is maintained by the implementation of consensus protocol.
+A member list is maintained by an implementation of consensus protocol.
 
-The rank-assign scheme in FTLib can extract worker identification from the member list, such like address of each worker. This unique identification helps the rank-assign scheme to designate individual rank number to each worker, which most communication libraries require when initializing.
+The rank-assign scheme in FTLib can extract worker identification from the member list, such like address of each worker, which helps the rank-assign scheme to designate individual rank number to each worker.
 
-When FTLib start to `rebuild`, it uses the `confirm` API of consensus protocol to check the member list is agreed by all existing workers.
+As described in [`FTLib`](./ftlib.md), if active mode is on, the `confirm` API will be called for in `initialized()` and `skip_allreduce()`, consenting member list before communication API calls.
```

Comment:
Explain briefly what "consenting member list" implies

Reply:
I changed "consenting member list" to "checking if member list has changed".